### DISCUSSION TITLE
[Feature] Support for password autofill on mobile app

### DISCUSSION THREAD

**Discussion by codeinabox** [immich-app/immich#3618] [Feature Request]
### The feature

I use 1Password on Android, and I am only able to autofill the email address when logging in and not the password. As I often get logged out of the mobile app, this would be a very handy feature.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by etnoy** [comment]
Agreed, thanks for bringing this to our attention

**Comment by github-cli** [comment]
just for the record, same on ios, it seems as if there is no app url for any password management app to recognize the app with

**Comment by aviv926** [comment]
Tip: I wouldn't use Immich's default authentication option (assuming it's shared publicly on the internet) as there is no protection against password brute force attacks (and it's not going to be implemented anytime soon if [ever](https://github.com/immich-app/immich/discussions/5436#discussioncomment-7947168))

I would recommend configuring [OAuth ](https://immich.app/docs/administration/oauth)instead

**Comment by bo0tzz** [comment]
This should be working fine these days, does anybody still have issues with it?

**Reply by etnoy** [reply]
Yes, this works fine for me on Android. Feel free to close
